mac.md

```diff
@@ -93,7 +93,11 @@ The following information will be important for you to know about the installati
 
 **If you ever want to preview the application that is running within your coding environment** — you will need to...
 
-[STEPS HERE]
+  - Use the `Run` button provided in the browser environment. The IDE will provide you with a URL you can use to view your application:
+
+    <img width="600" alt="codenvy run button" src="images/codenvy/run.png" />
```

Comment:
Since we don't have other screenshots in these high level OS guides, I think to make it consistent, it might be best to remove this (as opposed to adding screenshots to other environments).  Just my 2-cents - open to thoughts here.

Reply:
That's fair - I think it makes it easier, but linking to the guide is probably sufficient.